frontend/src/libraries/explorer-wamp/transactions.ts

```diff
@@ -57,12 +57,17 @@ export interface Action {
   DeleteKey: DeleteKey;
 }
 
+export interface ActionWrapper {
+  kind: string;
```

Comment:
Is there any problem with defining it `keyof Action` type instead of generic string?

Reply:
changed back